src/tabs/ConstantsPage.js

```diff
@@ -30,25 +29,128 @@ import {
 } from '../components/button';
 import {ROTATION} from '../components/rotation.js';
 import {PROPOTION} from '../components/trigonometry';
+import {
+    GetConstantsList,
+    GetConstantsLabels,
+} from '../server_communication/constants_api';
+import {BTPostData,BTPostHex} from '../bt_communication/bt_data_sender';
+import { hex_to_ascii, int_to_ascii, int_to_hex } from '../utils/VariableFormat';
+import { constants_default_values } from '../utils/DefaultValues';
 
 export const ConstantsPage = props => {
+    const USE_ROBONITOR = true;
+
+    const [constantList, setConstantList] = useState(constants_default_values);
+
+    useEffect(() => {
+        async function getConstant() {
+            const constant = await GetConstantsLabels();
+            constant = constant == null ? constants_default_values : constant;
+            setConstantList(constant);
+        }
+        getConstant();
+    }, []);
+
+    const setConstantValue = () => {};
+
+    const ConstantInput = ({constant}) => {
+        // useState[]
+        return (
+            <View style={styles.tableCell}>
+                <View style={styles.textView}>
+                    <Body>{constant.description}</Body>
+                </View>
+
+                <View style={styles.textInputView}>
+                    <TextInput
+                        style={styles.textInput}
+                        onChangeText={e => {
+                            // (constant.value = e);
+                            var foundIndex = constantList.findIndex(
+                                x => x.id == constant.id,
+                            );
+                            // console.log(constantList[foundIndex].value);
+                            constantList[foundIndex].value = e;
+                        }}
+                    />
+                </View>
+            </View>
+        );
+    };
+
+    const enviarButtonHandler = () => {
+        console.log("enviarButtonHandler");
+        constantList.forEach(({id,description,value}) => {
+            if(USE_ROBONITOR){
+                sendOneDataRobonitor(id,value);
+            } else {
+                sendOneDataMonitracer(id,value);
+            }
+        })
+    };
+
+    const salvarButtonHandler = () => {
+        console.log("salvarButtonHandler");
+    };
+
+    const clearButtonHandler = () => {
+        console.log("clearButtonHandler");
+        constantList.map((val,index) => {
```

Comment:
Você n devia usar `map` pra modificar valores, nesse caso que é um array de objetos, um for é suficiente

```js
for (const v of constantList) {
    v.value = 0
}
```